plugins/foreman_discovery/4.1/index.md

```diff
@@ -1,7 +1,18 @@
 ---
 layout: plugin
+pluginname: foreman_discovery
 title: Foreman Discovery 4.1 Manual
 version: 4.1
+# versions for matrix and snippets
+# (use short version for imgver e.g. 3.0)
+pluginver: 4.1.1
+proxyver: 1.0.3
+imgver: 3.0
+cliver: 0.0.2
+# uncomment to show warning box for an old release
+#warning: old
+# uncomment to show development version warning
+#warning: unreleased
```

Comment:
Could you put this into README.md instead please?  We'll forget it's here I think.